app/index.js

```diff
@@ -48,20 +48,25 @@ const url = 'file://' + resolve(
 console.log('electron will open', url);
 
 app.on('ready', () => {
-  function createWindow (fn) {
+  function createWindow (fn, options = {defaults: {position: undefined, size: undefined}}) {
```

Comment:
I agree, Just testing if options.position or options.size if undefined is better than using default.